### PR TITLE
Remove seven-ten

### PR DESCRIPTION
**Pull request by eatyourgreens**
Staging branch URL: https://pr-5952.pfe-preview.zooniverse.org

I've removed `seven-ten` wherever it was used in the code. Where visibility or text splits were used, I've reverting to showing the hidden content. That needs careful checking, in case there's text, eg. for workflow assignment, that  should be completely removed too.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/components/workflow-assignment-dialog.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import Dialog from 'modal-form/dialog';
 import ReactDOM from 'react-dom';
-import { TextSplit } from 'seven-ten';
 import Translate from 'react-translate-component';
 
 class WorkflowAssignmentDialog extends React.Component {
```
Comment (eatyourgreens) [inline app/components/workflow-assignment-dialog.jsx:7]: With `seven-ten` gone, I think we may be able to delete this entire component.

Comment (srallen) [reply app/components/workflow-assignment-dialog.jsx:7]: Nope. This is used by Gravity Spy and informs volunteers when they have a new workflow assignment giving them an option to load the new workflow or continue on the current workflow.

**Review by srallen — APPROVED** [review]
LGTM. Clarifying that `workflow_assignment` is the Gravity Spy feature now being generalized into FEM. They ran or was going to run several splits, some which didn't or ran but didn't get repeated, so I think this is all cleaned up here, but the assignment dialog should remain.

**Comment by eatyourgreens** [comment]
Cool. Let's get this deployed.

**Pull request merged**